controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -1101,6 +1140,15 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
           };
         }
 
+        for (const graph of updatedFederatedGraphs) {
+          orgWebhooks.send('graph.schema.updated', {
```

Comment:
Reminder: Send should not block. Does it?

Reply:
I dont await it, so it doesnt block

Reply:
Great